database/models.py

```diff
@@ -52,6 +52,7 @@ class HarvestSource(db.Model):
         index=True,
     )
     url = db.Column(db.String, nullable=False, unique=True)
+    size = db.Column(Enum("small", "medium", "large", name="size"))
```

Comment:
I guess with this being an enum the odds are tiny that bad input would get passed in...

Reply:
Yeah, that's what I was figuring.